monai/apps/auto3dseg/data_analyzer.py

```diff
@@ -169,6 +173,58 @@ def _check_data_uniformity(keys: list[str], result: dict) -> bool:
         return True
 
     def get_all_case_stats(self, key="training", transform_list=None):
+        """ Wrapper for the internal _get_all_case_stats to perform multi-gpu processing
```

Comment:
The docstrings of `get_all_case_stats` and `_get_all_case_stats` need to be updated.